mine_that_text.py

```diff
@@ -0,0 +1,173 @@
+"""
+Software Design Project 3
+Analyzing writing style in the Inheritance Cycle
+Anil Patel
+
+This project will be attempting to perform a variety of linguistic analyses on
+two books in the Inheritrance Cycle - Eragon, the first book, and Inheritance,
+the last book. I'm curious about the differences or similarities that might
+exist because the first book was written when the author was 15 years old and
+the last when he was 23.
+"""
+
+# Import some stuff
+import nltk
+
+# Setup books as lists with NLTK
+
+# Open Eragon sample into a list of words
+f = open("eragon_full.txt", "r")            # Open txt file
+eragon = f.read()                           # Convert into workable variable
+list_eragon = nltk.word_tokenize(eragon)    # Use NLTK to tokenize the text
+# print(list_eragon)                        # Test above line by printing
+
+
+# Do the same with inheritance sample
+g = open("inheritance_full.txt", "r")
+inheritance = g.read()
+list_inheritance = nltk.word_tokenize(inheritance)
+# print(list_inheritance)
+
+
+def fix_txt(some_list):
+    """
+    Function to separate word from random characters and end of sentence
+    punctuation that is misprinted in the txt file or parsed incorrectly by
+    NLTK
+
+    Separates end of sentence punctuation accidentally included in word strings
+    by NLTK so that I can analyze sentence lengths later.
+
```

Comment:
In a docstring describing a function, you usually include what parameter it takes and what kind of output it produces,